.env.example

```diff
@@ -1,4 +1,2 @@
-ETHERSCAN_API_KEY=ESPTDCMJIN1M4PX1V8EHJ4H6GIAE1D8HRI
-ROPSTEN_URL=https://ropsten.infura.io/v3/PROJECTID
-KOVAN_URL=https://kovan.infura.io/v3/PROJECTID
+# optional ETHERSCAN_API_KEY=ESPTDCMJIN1M4PX1V8EHJ4H6GIAE1D8HRI
```

Comment:
i would remove it entirely this line and etherscan references if only deploying in rsk.
If we wanted to keep it i will leave it but with and empty value

Reply:
changed.